### PR TITLE
Added Datasaur helm chart v/1.2.2

### PR DESCRIPTION
**Pull request by daniel-jeremia**

### CODE REVIEW

File: charts/datasaur/Chart.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.2.0
+version: 1.2.1
```
Comment (tonoman3g) [inline charts/datasaur/Chart.yaml]: bump to 1.3.0, because we added new services for intelligence

File: charts/datasaur/values.yaml [hunk 2]
```diff
@@ -97,14 +97,22 @@ services:
...
     enabled: true
-  languageCrunch:
-    enabled: true
   presidioAnalyzer:
     enabled: true
   rabbitmq:
     enabled: false
   redis:
     enabled: false
+  languageCrunch:
```
Comment (tonoman3g) [inline charts/datasaur/values.yaml:106]: where is the handling for these variables?

```
ENABLE_CORENLP_AUTO_LABEL=false
ENABLE_DISTILBERT_AUTO_LABEL=true
ENABLE_NLTK_AUTO_LABEL=false
ENABLE_SPACY_AUTO_LABEL=true
ENABLE_SPARKNLP_AUTO_LABEL=false
```

File: charts/datasaur/Chart.yaml (hunk 1 above)
Comment (daniel-jeremia) [reply charts/datasaur/Chart.yaml]: i already bump this from 1.1.0 to 1.2.0 when updating the spacy one @tonoman3g

Comment (tonoman3g) [reply charts/datasaur/Chart.yaml]: oh noted @daniel-jeremia

**Review by tonoman3g — APPROVED** [review]

**Pull request merged**